src/constants.js

```diff
@@ -22,6 +22,10 @@ export const FATHOM_CHANGELOG = process.env.GATSBY_FATHOM_CHANGELOG
 export const FATHOM_MADE_BY = process.env.GATSBY_FATHOM_MADE_BY
 export const FATHOM_SUPPORT = process.env.GATSBY_FATHOM_SUPPORT
 export const FATHOM_EXPORT = process.env.GATSBY_FATHOM_EXPORT
+export const FATHOM_DELETE_ALL_MENSES_TAGS = process.env.GATSBY_FATHOM_DELETE_ALL_MENSES_TAGS
+export const FATHOM_DELETE_TAGS_CONFIRMED = process.env.GATSBY_FATHOM_DELETE_TAGS_CONFIRMED
```

Comment:
**suggestion:** No need for confirmed on this as there is no confirmation process as it is for delete entries.

Reply:
suggestion: No need for confirmed #304
b0a52c8